toolchain/check/handle_binding_pattern.cpp

```diff
@@ -72,6 +69,17 @@ auto HandleBindingPattern(Context& context, Parse::BindingPatternId parse_node)
               context.node_stack().PeekParseNodeKind()) {
     case Parse::NodeKind::ReturnedModifier:
     case Parse::NodeKind::VariableIntroducer: {
+      if (is_generic) {
+        CARBON_DIAGNOSTIC(
+            GenericBindingInVarDecl, Error,
+            "`var` declaration cannot declare a generic binding.");
+        context.emitter().Emit(type_node, GenericBindingInVarDecl);
```

Comment:
Similar to the bool renaming:

```suggestion
            CompTimeBindingInVarDecl, Error,
            "`var` declaration cannot declare a compile-time binding.");
        context.emitter().Emit(type_node, CompTimeBindingInVarDecl);
```

Also, should this be `CompTime`, `CompileTime`, or `Comptime`? I have no opinion really, happy for you to pick whichever, but may be reflected in the name of the bool as well.

Reply:
Done.